core/src/main/kotlin/org/evomaster/core/problem/graphql/KindX.kt

```diff
@@ -0,0 +1,27 @@
+package org.evomaster.core.problem.graphql
+
+import org.evomaster.core.problem.graphql.schema.InputValue
+import org.evomaster.core.problem.graphql.schema.__Field
+
+class KindX {
```

Comment:
instead of separated class `QuadComponent`, just use KindX, with 2 extra custom constructor

Reply:
and aslo add documentation to specify what this `KindX` is for

Reply:
Done